input/fsh/moped-operation-verlegen.fsh

```diff
@@ -21,7 +21,7 @@ Die Operation wird vom Akteur Krankenhaus (KH) aufgerufen. Die $verlegen Operati
   * *MOPEDTransferEncounter.serviceProvider* setzt eine Referenz auf die MOPEDOrganizationAbteilung mit dem jeweiligen *funktionscode* bzw. *funktionssubcode* lt. Operation-Parameter.
   * *MOPEDTransferEncounter.Neugeborenes* wird lt. LKF-Regeln berechnet, anhand des *MOPEDEncounter.subject.birthdate* aus dem Encounter aus Schritt 1 (für Berechnugns-Details siehe Hinweis 1).
   * *MOPEDTransferEncounter.Altersgruppe* wird lt. LKF-Regeln berechnet, anhand des *MOPEDEncounter.subject.birthdate* aus dem Encounter aus Schritt 1 (für Berechnugns-Details siehe Hinweis 2).
-  * *MOPEDTransferEncounter.anwesenheitsart* enthält wert aus dem ValueSet *MOPEDTransferEncounter.subjectStatus.coding.display*
+  * *MOPEDTransferEncounter.anwesenheitsart* enthält einen code aus dem ValueSet *MOPEDTransferEncounter.subjectStatus*, danach soll andand des codes die beiden Felder *coding.display* und *coding.code* befüllt werden
```

Comment:
Die Anwesenheitsart gibt es so dann als Element im MOPEDTransferEncounter nicht mehr. Ich würde es so beschreiben:

* *MOPEDTransferEncounter.subjectStatus.code* enthält den code lt. Operations-Parameter `anwesenheitsart`, und zusätzlich wird aus dem ValueSet der Display-Wert ausgelesen und in *MOPEDTransferEncounter.subjectStatus.display* gepseichert.

Reply:
Ah ok jetzt hab ich's verstanden, habs nochmal geändert :D